tests/e2e/simple.spec.ts

```diff
@@ -165,3 +165,53 @@ test('does not monetize when continuous payments are disabled', async ({
     });
   });
 });
+
+test('does not monetize when toggle payments in extension is checked', async ({
+  page,
+  popup,
+  i18n,
+}) => {
+  const walletAddressUrl = process.env.TEST_WALLET_ADDRESS_URL;
+  const playgroundUrl = 'https://webmonetization.org/play/';
+
+  await test.step('disables extension', async () => {
+    await popup.locator('label:has(> input[type="checkbox"])').click();
+
+    await expect(popup.locator('h3')).toHaveText(
+      i18n.getMessage('app_text_disabled'),
+    );
+  });
+
+  await page.goto(playgroundUrl);
+
+  const monetizationCallback = spy<[Event], void>();
+  await page.exposeFunction('monetizationCallback', monetizationCallback);
+  await page.evaluate(() => {
+    window.addEventListener('monetization', monetizationCallback);
+  });
+
+  await test.step('check extension payments do not go through', async () => {
+    await page
+      .getByLabel('Wallet address/Payment pointer')
+      .fill(walletAddressUrl);
+    await page.getByRole('button', { name: 'Add monetization link' }).click();
+
+    await expect(page.locator('link[rel=monetization]')).toHaveAttribute(
+      'href',
+      walletAddressUrl,
+    );
+
+    await expect(
+      popup.getByRole('button', { name: 'Send now' }),
+    ).not.toBeVisible();
+    await page.waitForTimeout(1000);
+    await expect(monetizationCallback).toHaveBeenCalledTimes(0, { wait: 2000 });
+  });
+
```

Comment:
Let's add a case where we toggle continuous payments off/on in Settings > Rate, and ensure there's no monetization event still.